packages/react-native-payments/README.md

```diff
@@ -0,0 +1,94 @@
+# @farfetch/blackout-react-native-payments
+
+Payments solutions for react-native apps
+
+## Installation
+
+**yarn**
+
+```sh
+yarn add @farfetch/blackout-react-native-payments
+```
+
+**npm**
+
+```sh
+npm i @farfetch/blackout-react-native-payments
+```
+
+### Peer dependencies
+
+Make sure that you have installed the correct peer dependencies of this package:
+
+- [`@farfetch/blackout-react`](https://www.npmjs.com/package/@farfetch/blackout-react)
+- [`react-native-webview`](https://www.npmjs.com/package/react-native-webview)
+
+## Usage
+
+```js
+import {
+  CreditCard,
+  creditCardinitialState,
+  creditCardreducer,
+  OPERATION_TYPE,
+} from '@farfetch/blackout-react-native-payments';
+// ...
+// ... other imports
+// ...
+
+const FOLDER_NAME = '';
+const LOCALE = '';
+const STATIC_NAME = '';
+const PAYMENT_GATEWAY_URL = '';
+
+const OPERATION = JSON.stringify({
+  action: OPERATION_TYPE.ADD_CREDIT_CARD_INSTRUMENT,
+});
+
+const CREATE_INSTRUMENT_JS = `window.postMessage(${OPERATION}, "${PAYMENT_GATEWAY_URL}")`;
+
+const Checkout: React.FC = () => {
+  const webViewRef = useRef();
+
+  const [creditCardState, creditCardDispatch] = useReducer(
+    creditCardreducer,
+    creditCardinitialState,
+  );
+
+  const handleCharge = (): void => {
+    if (creditCardState.isFormValid) {
+      webViewRef.current?.injectJavaScript(CREATE_INSTRUMENT_JS);
+    }
+  };
+
+  return (
+      <CreditCard
+        additionalProperies={{
+          folderName: FOLDER_NAME,
+          locale: LOCALE,
+          staticName: STATIC_NAME,
+          url: PAYMENT_GATEWAY_URL,
+        }}
+        creditCardDispatch={creditCardDispatch}
+        paymentIntentId={checkoutOrder.paymentIntentId}
+        webViewRef={webViewRef}
+      />
+      <S.ButtonView>
+        <Button fullwidth primary onPress={handleCharge}>
+          {t('submit')}
```

Comment:
```suggestion
          submit
```

Nitpick: for the sake of the example I'd remove the translation function 🙂

Reply:
Should be ok now, please double check 👍